src/components/Calculator.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
- [ ] Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.

Reply:
I don't know how to solve this eslint error.